### ISSUE TITLE
5.1.1 update

### ISSUE THREAD

**Issue by subjekt-iv** [GetMetaMap/metamap-reactnative-plugin#79]
Hey folks! I'm encountering this issue when trying to build the app for IOS, tried pod repo update and pod install --repo-update but no luck. Any suggestions?

[!] CocoaPods could not find compatible versions for pod "MetaMapSDK":
  In Podfile:
    react-native-metaMap-sdk (from `/Users/Agustin/Documents/brik-mobile-app/brik-native/node_modules/react-native-metamap-sdk/react-native-metaMap-sdk.podspec`) was resolved to 5.1.1, which depends on
      MetaMapSDK (~> 3.16.1)

None of your spec sources contain a spec satisfying the dependency: `MetaMapSDK (~> 3.16.1)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

**Comment by subjekt-iv** [comment]
hey @AvoSukiasyan could you check this out? thanks in advance!

**Comment by AvoSukiasyan** [comment]
hey @subjekt-iv . 
I created an example RN project and added our React-Native plugin. updated plugin version to 5.1.1 latest version and iOS SDK (pod version) 3.16.1 and the latest version. for me works as expected. can you update the plugin and try again, please?
if it will not help, please send me your interaction part/ code. maybe the problem related to the integration.

**Comment by subjekt-iv** [comment]
will try it then, thanks for getting back to me!

**Comment by subjekt-iv** [comment]
Hey @AvoSukiasyan!

Sorry about the delay; I've been intensely working on Android and am now focusing on iOS. I was able to fix the issue I was facing. It was due to my machine (MBP M2 chip). I couldn't run "pod install --repo-update" because I constantly encountered an error. However, thanks to this solution (https://stackoverflow.com/questions/67443265/error-regarding-undefined-method-map-for-nilnilclass-for-flutter-app-cocoap), I was able to update the SDK to the latest version. Thank you very much for your help!
